dune_client/viz/graphs.py

```diff
@@ -0,0 +1,83 @@
+"""
+Functions you can call to make different graphs
+"""
+
+import plotly.graph_objects as go
+import colorlover as cl
+import pandas as pd
```

Comment:
You're going to need to add all of these packages to the requirements/prod.txt file and also in setup.cfg:

https://github.com/duneanalytics/dune-client/blob/b510c974f2e826a0f6e72392b4988515550772a2/setup.cfg#L22

Reply:
ah thank you! I was so confused why the build kept failing here! decided to go with the dev route!